### ISSUE TITLE
Neural Net Doesn't Work on Rotated MNIST

### ISSUE THREAD

**Issue by windfreedom** [zyxue/stanford-cs229#18]
https://github.com/zyxue/stanford-cs229/blob/master/Problem-set-4/1-Neural-networks-MNIST-image-classification.ipynb

When I try to train your implementation on a version of MNIST we were given by our teachers as a "sanity check" for me, it doesn't work. Our MNIST comes from Kaggle and (when I visualized it) is the same size as yours, only rotated 90 degrees. I push my MNIST dataset through your pipeline without modification and the accuracy is 0 after 10 epochs of training.

**Comment by zyxue** [comment]
Do you have a notebook that I could take a look? Also, could you provide a url to the dataset you are referring to?

**Comment by zyxue** [comment]
Closed for inactivity, feel free to reopen if needed.
